packages/next/client/experimental-script.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect, useContext } from 'react'
+import { ScriptHTMLAttributes } from 'react'
+import { HeadManagerContext } from '../next-server/lib/head-manager-context'
+
+const ScriptCache = new Map()
+const LoadCache = new Set()
+
+const DOMAttributeNames: Record<string, string> = {
+  acceptCharset: 'accept-charset',
+  className: 'class',
+  htmlFor: 'for',
+  httpEquiv: 'http-equiv',
+}
+
+interface Props extends ScriptHTMLAttributes<HTMLScriptElement> {
+  strategy?: 'defer' | 'lazy' | 'dangerousRenderBlocking' | 'eager'
```

Comment:
Let's please rename this:
```suggestion
  strategy?: 'defer' | 'lazy' | 'dangerouslyBlockRendering' | 'eager'
```